### PR TITLE
Lavaland welcoming billboard

### PR DESCRIPTION
**Pull request by McRamon**
## What Does This PR Do
Ads a billboard in front of the minin shuttle with  a randomized caption.

## Why It's Good For The Game
Lavaland is a comically lethal workplace with little to no safety measures and a mix of things happening on it. This purposefully out of place billboard enchances the cartoonish vibe of the game and greets arriving miners with a funny caption.

## Images of changes
dreamseeker_Hb8ZVxFewp
Photoshop_F5dkcAr3Ms

Sprite inspired by Viro's spritework (discord - itsviro)
## Testing

Tested locally for it to work as intended.

## Changelog
:cl: McRamon, GDN
add: New billboard on mining station
/:cl:

### CODE REVIEW

File: code/game/objects/structures/lavaland/billboard.dm [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+	icon_state = "billboard"
+	bound_width = 64
+	bound_height = 32
+	pixel_x = -2
+	anchored = TRUE
+	density = TRUE
+	max_integrity = 400
+	armor = list(MELEE = 20, BULLET = 20, LASER = 20, ENERGY = 100, BOMB = 0, RAD = 0, FIRE = 50, ACID = 50)
+	resistance_flags = LAVA_PROOF | FIRE_PROOF
+	var/list/billboard_overlay=list("chasmland","incompatible","feed","warm","adventure","plasmaland","gate","ufo","notfriendly","gps","service","protection","diamonds","step")
```
Comment (Henri215) [inline code/game/objects/structures/lavaland/billboard.dm:13]:
```suggestion
	var/static/list/billboard_overlay = list("chasmland", "incompatible", "feed", "warm", "adventure", "plasmaland", "gate", "ufo", "notfriendly", "gps", "service", "protection", "diamonds", "step")
```
static list like GDN advised on discord

File: code/game/objects/structures/lavaland/billboard.dm [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+	layer = EDGED_TURF_LAYER
+	resistance_flags = FIRE_PROOF
+	blocks_emissive = FALSE
+
+/obj/structure/lavaland_billboard/Initialize()
+	. = ..()
+	add_overlay(mutable_appearance(/obj/structure/lavaland_billboard,pick(billboard_overlay)))
```
Comment (Henri215) [inline code/game/objects/structures/lavaland/billboard.dm:20]:
```suggestion
	add_overlay(mutable_appearance(/obj/structure/lavaland_billboard, pick(billboard_overlay)))
```

**Review by AffectedArc07 — APPROVED** [review]
I love it

**Review by DGamerL — APPROVED** [review]
This is amazing

File: code/game/objects/structures/lavaland/billboard.dm [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+	pixel_x = -2
+	anchored = TRUE
+	density = TRUE
+	max_integrity = 400
+	armor = list(MELEE = 20, BULLET = 20, LASER = 20, ENERGY = 100, BOMB = 0, RAD = 0, FIRE = 50, ACID = 50)
+	resistance_flags = LAVA_PROOF | FIRE_PROOF
+	var/static/list/billboard_overlay = list("chasmland", "incompatible", "feed", "warm", "adventure", "plasmaland", "gate", "ufo", "notfriendly", "gps", "service", "protection", "diamonds", "step")
+	layer = EDGED_TURF_LAYER
+	resistance_flags = FIRE_PROOF
+	blocks_emissive = FALSE
```
Comment (Henri215) [inline code/game/objects/structures/lavaland/billboard.dm:16]:
```suggestion
	layer = EDGED_TURF_LAYER
	resistance_flags = FIRE_PROOF
	blocks_emissive = FALSE
	var/static/list/billboard_overlay = list("chasmland", "incompatible", "feed", "warm", "adventure", "plasmaland", "gate", "ufo", "notfriendly", "gps", "service", "protection", "diamonds", "step")
```
We usually put new vars at the end

**Review by Henri215 — APPROVED** [review]
Lewcing good

**Comment by Gaxeer** [comment]
Looks great

**Comment by Pedritoc25** [comment]
This is great

**Comment by Pedritoc25** [comment]
It looks nice. Ive got a question. What happens if you build a dock in a free golem suttle dock or any other white shuttle dock?

**Pull request merged**